src/components/Perils/data/usePerils.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import { Peril, TypeOfContract } from '../types'
+
+export const usePerils = (insuranceType: TypeOfContract) => {
+  const [perils, setPerils] = useState<[] | Peril[]>([])
+
+  const fetchPerils = async () => {
+    const url = `https://graphql.dev.hedvigit.com/graphql`
+
+    const data = {
+      operationName: 'Perils',
+      variables: {
+        typeOfContract: insuranceType,
+        locale: 'sv_SE',
```

Comment:
One more thing i realised - we shouldn't forget to change this. We could probably use the [`Context`'s](https://github.com/HedvigInsurance/web-next/blob/master/src/components/containers/ContextContainer.tsx) `lang` property

Reply:
Word, I'll fix that in the next PR :)